libs/eo/landing-page/shell/src/lib/eo-landing-page-header.component.ts

```diff
@@ -57,12 +57,14 @@ const selector = 'eo-landing-page-header';
           background: var(--watt-color-primary);
           color: var(--watt-color-primary-contrast);
 
-          // This is equivalent to a normal size watt button
+          // This is equivalent to a normal size watt button width
           min-width: 6.25rem;
```

Comment:
suggestion: inline the comment so that it's clear that it's not for the whole section of properties. Alternatively, use a named Sass or CSS variable to make intent clear.
```suggestion
          min-width: 6.25rem; // This is equivalent to a normal size watt button width
```